src/main/docker/NOTES.MD

```diff
@@ -0,0 +1,44 @@
+Some Commands to run and work with the Docker image
+
+- Prepare the code and build the project
+
+```console
+mvn fmt:format
```

Comment:
`./mvnw fmt:format` ideally to use the exact version of Maven CI uses (instead of whatever system happens to have)